src/feature/nodelist/networkstatus.c

```diff
@@ -1604,7 +1605,9 @@ networkstatus_consensus_has_ipv6(const or_options_t* options)
 }
 
 /** Given two router status entries for the same router identity, return 1 if
- * if the contents have changed between them. Otherwise, return 0. */
+ * if the contents have changed between them. Otherwise, return 0.
+ * It also checks for changes that are output by control port*/
```

Comment:
```suggestion
 * It only checks for fields that are output by control port. */
```

Please also add a comment to these 3 places, saying that these 3 places must be kept in sync:
* the struct routerstatus_t,
* the control port printing function routerstatus_format_entry() in NS_CONTROL_PORT mode, and
* this function.

See:
https://github.com/torproject/tor/blob/master/src/feature/nodelist/routerstatus_st.h#L19
https://github.com/torproject/tor/blob/master/src/feature/nodelist/fmt_routerstatus.c#L47

It might also help to read:
https://github.com/torproject/tor/blob/master/src/feature/nodelist/networkstatus.c#L2360